config/routes.rb

```diff
@@ -13,7 +13,7 @@
   resources :merchants, only: :show do
     get "/dashboard", to: "merchants#show"
 
-    resources :items, only: [:index, :show, :edit, :update], controller: "merchant_items"
+    resources :items, only: [:index, :show, :edit, :update, :new, :create], controller: "merchant_items"
```

Comment:
Would this be easier because delete is the only function we don't have?
```suggestion
    resources :items, except: [:delete], controller: "merchant_items"
```